### PR TITLE
Add Ireland high level decommission plan for production

### PR DESCRIPTION
**Pull request by corlettb**
What
----

Add Ireland high level decommission plan for production

How to review
-------------

Read the changes.

Who can review
--------------

Any PaaS team member

### CODE REVIEW

File: source/upcoming_plans/ireland_decomissioning.html.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+## Post Decommissioning Clean Up
+
+- [ ] Update documentation to remove references to the Ireland production environment. This includes:
+    - [ ] [team-manual](https://github.com/alphagov/paas-team-manual)
+    - [ ] [product-pages](https://github.com/alphagov/paas-product-pages)
+    - [ ] [paas-tech-docs](https://github.com/alphagov/paas-tech-docs)
+- [ ] Remove environment config in [paas-credentials](https://github.com/alphagov/paas-credentials).
+- [ ] Remove references to prod environment (not the prod account) in [paas-cf](https://github.com/alphagov/paas-cf) and [paas-bootstrap](https://github.com/alphagov/paas-bootstrap). We have a 'prod' and 'prod-lon' environment that both use the 'prod' aws account.
+- [ ] Remove alerting for ireland in Pagerduty
```
Comment (dark5un) [inline source/upcoming_plans/ireland_decomissioning.html.md:97]: Should alerting of Ireland on Pagerduty be removed first?

Comment (corlettb) [reply source/upcoming_plans/ireland_decomissioning.html.md:97]: Do we actually have any ireland specific config in pagerduty?

Comment (kr8n3r) [reply source/upcoming_plans/ireland_decomissioning.html.md:97]: do each these count as ireland specific. example https://governmentdigitalservice.pagerduty.com/service-directory/P2822RP/integrations has 
prod/missing_logs
prod-lon/missing_logs
Screenshot 2024-03-11 at 12 52 04

I assume that would need to be deleted from PD

Comment (corlettb) [reply source/upcoming_plans/ireland_decomissioning.html.md:97]: Yeah okay. I think those are used to "deduplicate" alerts coming from elasticsearch. We can clean them up after the decommission. However might make sense to decommission the logit missing log alert before the decommission begins.

Comment (DominicGriffin) [reply source/upcoming_plans/ireland_decomissioning.html.md:97]: Added this note to explore for another merge to the team manual

**Pull request merged**